gulp/tasks/copy.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 var gulp = require( 'gulp' );
-var $ = require( 'gulp-load-plugins' )();
-var config = require( '../config' ).copy;
-var handleErrors = require( '../utils/handleErrors' );
+var plugins = require( 'gulp-load-plugins' )();
```

Comment:
I would get behind `gulpLoadPlugins` too, as it describes the package name better.. but the original issue said `plugins` and that's at least clearer than `$`.